stix_shifter_modules/tanium/configuration/lang_en.json

```diff
@@ -0,0 +1,31 @@
+{
+    "connection": {
+        "host": 
+        {
+            "label": "Host IP address or hostname",
+            "description": "Specify the IP address or hostname of the data source"
+        },
+        "port": 
+        {
+            "label": "Host port",
+            "description": "Set the port number that is associated with the hostname or IP address."
+        },
+        "options": {
```

Comment:
you can remove this "options" block since it is already defined in common stix_shifter_modules/config.json